src/diskquota.h

```diff
@@ -319,4 +327,6 @@ extern HTAB *DiskquotaShmemInitHash(const char *name, long init_size, long max_s
 extern void  refresh_monitored_dbid_cache(void);
 extern HASHACTION check_hash_fullness(HTAB *hashp, int max_size, const char *warning_message,
                                       TimestampTz *last_overflow_report);
+void              SPI_connect_my(SPI_state *state);
+void              SPI_finish_my(const SPI_state *state);
```

Comment:
We should probably rename these to something like `SPI_connect_diskquota` or `SPI_connect_wrapper` instead

Reply:
renamed